### PR TITLE
Don't run eslint on build

### PR DESCRIPTION
**Pull request by jnields**

### CODE REVIEW

**Comment by DebugXYZ** [comment]
We should create an issue describing the problem that this is trying to solve so a higher level conversation can happen there.

**Comment by DebugXYZ** [comment]
Possibly related issues

https://github.com/rei/febs/issues/43
https://github.com/rei/febs/issues/44

**Comment by peripateticus** [comment]
This is also related to #41

File: webpack-config/webpack.base.conf.js [hunk 1]
```diff
@@ -85,24 +85,6 @@ module.exports = {
           },
         },
       },
-      {
```
Comment (peripateticus) [inline webpack-config/webpack.base.conf.js]: Yeah, there are some unit tests around linting which are failing so we'll need to remove those tests. I saw some other failures that didn't seem to be related to linting but I'll need to take a closer look.

**Comment by peripateticus** [comment]
I'm going to close this PR as I need to make additional updates related to removing eslint.

**Pull request closed without merging**